### DISCUSSION TITLE
I tried building the 6.12.0-rc5 kernel on Orange pi 5.

### DISCUSSION THREAD

**Discussion by hakotani-o** [Joshua-Riek/ubuntu-rockchip#1115] [Ideas]
I tried building the 6.12.0-rc5 kernel on Orange pi 5.
![2024-10-31 23-18-26](https://github.com/user-attachments/assets/149fd34f-578d-428d-8a37-c6d42b2acaa6)

![2024-10-31 23-18-43](https://github.com/user-attachments/assets/fbb18430-c006-4b46-9f3f-e1fd5093aefa)

I'm glad it booted successfully.

**Comment by nrpetonr** [comment]
nice. how is hdmi on this version?

**Reply by hakotani-o** [reply]
Today , can not boot.
 disappointing.
I have to order a new opi5.

**Reply by hakotani-o** [reply]
Hi nrpetonr.

Opi5 has been revived.
The source code is linux-next-next-20241101.tar.gz.
Today i build it.
What command should I run to check the hdmi version?
For now, I'll attach the glxinfo output.
[glxinfo.txt](https://github.com/user-attachments/files/17605537/glxinfo.txt)

Thank you for your support.

**Reply by nrpetonr** [reply]
my question was if it otputs anything to your display

**Reply by hakotani-o** [reply]
Sorry.
I write this message on 6.12.0-rc5-rockchip.
![2024-11-03 08-48-11](https://github.com/user-attachments/assets/ba4538a7-6184-4884-a9d5-f00942ce4ce1)
Thank you.

**Reply by Fredrum** [reply]
Very interesting! Thank you for showing!
Are you getting Audio-out over the hdmi?

**Reply by hakotani-o** [reply]
Audio not work.
![2024-11-04 12-47-09](https://github.com/user-attachments/assets/5fc962ad-f1b5-4eb1-b256-ae90b4a9ccda)
USB-PH output only left.
Audio doesn't work well even with 6.11.

Thank you.

**Reply by rivolity** [reply]
Can you mention all steps you have followed to build the kernel 6.12 ? Please

**Reply by hakotani-o** [reply]
Thank you for your interest.
Currently, I think the 6.13 kernel is more timely.
We apologize for the inconvenience, but please refer to the link below.
P.S. The build script is also attached.

https://github.com/Joshua-Riek/ubuntu-rockchip/discussions/1135

Thank you.
